### PR TITLE
screen share aspect ratio fix

### PR DESCRIPTION
**Pull request by jashan1498**

### CODE REVIEW

File: android/src/main/kotlin/live/hms/hmssdk_flutter/views/HMSVideoView.kt [hunk 1]
```diff
@@ -25,12 +33,89 @@ class HMSVideoView(context: Context,setMirror:Boolean) : ConstraintLayout(contex
 
         surfaceViewRenderer = view.findViewById(R.id.surfaceViewRenderer)
         surfaceViewRenderer.setEnableHardwareScaler(true)
-        Log.i("HMSVideoViewAndroid",setMirror.toString())
+        Log.i("HMSVideoViewAndroid", setMirror.toString())
         surfaceViewRenderer.setMirror(setMirror)
-        surfaceViewRenderer.setScalingType(RendererCommon.ScalingType.SCALE_ASPECT_FILL)
+        surfaceViewRenderer.setScalingType(RendererCommon.ScalingType.SCALE_ASPECT_FIT)
```
Comment (ygit) [inline android/src/main/kotlin/live/hms/hmssdk_flutter/views/HMSVideoView.kt:38]: should come from clients, default to `SCALE_ASPECT_FIT`

File: lib/src/ui/meeting/hms_video_view.dart [hunk 2]
```diff
@@ -18,9 +18,14 @@ class HMSVideoView extends StatelessWidget {
   /// [HMSVideoView] will use viewSize to get height and width of rendered video. If not passed, it will take whatever size is available to the widget.
   final Size? viewSize;
   bool setMirror;
+  final bool isScreenShare;
```
Comment (ygit) [inline lib/src/ui/meeting/hms_video_view.dart:21]: this info can be taken from the HMSTrack's source property

**Review by ygit — CHANGES_REQUESTED** [review]

File: lib/src/ui/meeting/hms_video_view.dart (hunk 2 above)
Comment (jashan1498) [reply lib/src/ui/meeting/hms_video_view.dart]: resolved

File: android/src/main/kotlin/live/hms/hmssdk_flutter/views/HMSVideoView.kt (hunk 1 above)
Comment (jashan1498) [reply android/src/main/kotlin/live/hms/hmssdk_flutter/views/HMSVideoView.kt]: resolved

File: lib/src/ui/meeting/hms_video_view.dart (hunk 2 above)
Comment (Arunshaik2001) [reply lib/src/ui/meeting/hms_video_view.dart]: hms_video_scale_type is missing

Comment (jashan1498) [reply lib/src/ui/meeting/hms_video_view.dart]: resolved

**Review by ygit — APPROVED** [review]

**Pull request merged**